### DISCUSSION TITLE
How is transfer size calculated ?

### DISCUSSION THREAD

**Discussion by gaborkalmar** [GoogleChrome/lighthouse#13714] [Q&A]
Hey ! I'm looking to improve the LH score of a project and in the network tab I'm getting 243kb as the size of the bundle, but on LH it appears as 900kb transfer size, with 500kb unused. How is this calculated ? Where should I even start with this ?

**Comment by adamraine** [comment]
Lighthouse transfer size should be calculated from the same requests as the network panel, a difference that large seems weird. Can you share an example report or the URL you are testing?

**Reply by gaborkalmar** [reply]
Hey, yea sure, [this](https://weaintgotnohistory.sbnation.com/chelsea-fc-transfer-rumours-news/2022/3/3/22959550/marina-granovskaia-bruce-buck-unlikely-to-stay-at-chelsea-after-abramovich-report) , or [this](https://www.wral.com/man-dies-after-being-hit-by-car-in-garner/20166206/). The connatix stuff is what i'm looking at, `connatix.playspace.js` , it's a big bundle I know, but not 'that' big, that's our issue mostly.

**Reply by gaborkalmar** [reply]
I found some interesting things here. When we request our script we do a redirect based on user agent, to detect es5 or european bundles, the numbers that show up in lighthouse seem to be the size of our bundle uncompressed, but in the network it shows as it's served compressed ( br ) . Is it possible that this redirect 'confuses' the lighthouse measurements ?

**Reply by adamraine** [reply]
>  Is it possible that this redirect 'confuses' the lighthouse measurements ?

Yes this is a bug #13724
